lib/kafka_ex/utils/murmur.ex

```diff
@@ -0,0 +1,70 @@
+defmodule KafkaEx.Utils.Murmur do
+  @moduledoc """
+  Utility module that provides Murmur hashing algorithm.
+  """
+
+  use Bitwise
+
+  # Arbitrary constant for murmur2 hashing
+  # https://github.com/aappleby/smhasher/blob/master/src/MurmurHash2.cpp#L39-L43
+  @m 0x5BD1E995
+  @r 24
+
+  # Default seed to hashing, copied form Java implementation
+  # https://github.com/apache/kafka/blob/2.1.0/clients/src/main/java/org/apache/kafka/common/utils/Utils.java#L382
```

Comment:
please update to:

https://github.com/apache/kafka/blob/809be928f1ae004e11d65c307ea322bef126c834/clients/src/main/java/org/apache/kafka/common/utils/Utils.java#L382

so it references the sha of the tag, if someone force pushes over the tag the line might change but it does not change with the sha